docs/higlass_server.rst

```diff
@@ -109,6 +109,67 @@ To modify a tileset name, specify the tileset `uuid` in the URL, use the `PATCH`
 
   curl --user ${username}:${password} --request PATCH --header "Content-Type: application/json" --data '{"name":"new_name_of_tileset"}' http://localhost:8000/api/v1/tilesets/${uuid}/
 
+To retrieve a particular tile, in this case tile `1` at zoom level `6` for the tileset named by the `${uuid}` variable:
+
+.. code-block:: bash
+
+  curl localhost:8000/api/v1/tiles?d=${uuid}.6.1
+
+To retrieve many tiles at once, in this case tiles `0`, `1`, and `2` at zoom level `6` for the tileset `${uuid}`:
```

Comment:
Did  you try rendering this? I think with rst you need double backticks to get code formatting.

You can try it out like this:

```
cd docs
./serve.sh
```

Reply:
No I did not. I just tried it and updated the formatting.